hibernate-reactive-core/src/test/java/org/hibernate/reactive/annotations/TestingUtil.java

```diff
@@ -0,0 +1,135 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive.annotations;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.platform.commons.support.AnnotationSupport;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author Steve Ebersole
+ * @see:  https://github.com/hibernate/hibernate-orm/hibernate-testing/src/main/java/org/hibernate/testing/orm/junit/TestingUtil.java
+ */
+public class TestingUtil {
```

Comment:
Do we need this?

Reply:
Not the whole thing. I'll need one of the methods so I'll just move it to DBTypeFilterExtension.